app/controllers/api/v1/rooms_controller.rb

```diff
@@ -39,6 +39,8 @@ def start
           join_url = bbb_api.start_meeting room: @room, meeting_starter: current_user, options: options
           logger.info "meeting successfully started for room(friendly_id):#{@room.friendly_id} by #{meeting_starter}."
 
+          ActionCable.server.broadcast "#{@room.friendly_id}_rooms_channel", join_url.to_s
```

Comment:
Broadcast the join url to everyone informing them that the room has started (needs to be changed in the next PR)